examples/sd_spi.rs

```diff
@@ -0,0 +1,63 @@
+#[cfg(esp32)]
+fn main() -> anyhow::Result<()> {
+    use esp_idf_hal::{
```

Comment:
Nit: `esp_idf_svc::hal` instead. This way we hint the user that she only needs to depend on `esp_idf_svc`, as it re-exports the hal and sys crates anyway.